e2e/tests/mcp-setup-wizard.spec.ts

```
@@ -0,0 +1,330 @@
+
+        // Step 2: Select existing route
+        await expect(page.locator('[data-test="mcp-route-select"]')).toBeVisible({ timeout: 15_000 });
+        await page.locator('[data-test="mcp-route-select"]').selectOption(routeName);
+        await nextButton.click();
+
+        // Step 3: Fill MCP Extension form
+        await expect(page.locator('[data-test="mcp-extension-name"]')).toBeVisible({ timeout: 15_000 });
+        await page.locator('[data-test="mcp-extension-name"]').fill('e2e-mcp-ext');
+
+        // Select listener from dropdown (gateway has 'http' listener)
+        const sectionSelect = page.locator('[data-test="mcp-section-name"]');
```

ESLint

[error] Delete `··`

(prettier/prettier)